pkg/scheduler/podname.go

```diff
@@ -0,0 +1,28 @@
+package scheduler
+
+import (
+	"fmt"
+	"github.com/kyma-incubator/octopus/pkg/apis/testing/v1alpha1"
+)
+
+const (
+	TestingPodPrefix = "oct-tp"
+)
+
+type PodNameGenerator struct{}
+
+func (P *PodNameGenerator) GetName(suite v1alpha1.ClusterTestSuite, def v1alpha1.TestDefinition) (string, error) {
+	// max length of k8s name is 253 characters
```

Comment:
but here you should ensure that generated name has max 253 chars. It's not case of validation is more about proper pattern for creating name.

Reply:
Currently, we concatenate 2 names so the problem can occur. My idea is to limit the length of a test suite and test definition to 120 characters. In addition to that, I will check length also here.